bin/compile

```diff
@@ -68,30 +68,44 @@ case "$stack" in
     # the package list is found by using ci:debug then running ldd $GOOGLE_CHROME_BIN | grep not
     # also look here for more packages/notes https://developers.google.com/web/tools/puppeteer/troubleshooting
     PACKAGES="
-      gconf-service
-      libappindicator1
+      ca-certificates
```

Comment:
I alphabetized this since that's what's on the puppeteer page. This diff is a bit noisier than it should be but hopefully future diffs should be much cleaner.